pkg/trace/local_tracer_test.go

```diff
@@ -0,0 +1,153 @@
+package trace
+
+import (
+	"fmt"
+	"io"
+	"net"
+	"os"
+	"path"
+	"testing"
+	"time"
+
+	"github.com/cometbft/cometbft/config"
+	"github.com/cometbft/cometbft/libs/log"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	// CannalTable is the table name for the Cannal struct.
+	CannalTable = "cannal"
+)
+
+type Cannal struct {
+	Ville    string `json:"city"`
+	Longueur int    `json:"length"`
```

Comment:
[blocking] I'm so confused 🤯 why are these struct fields in french  but the JSON is in English?

Reply:
tired evan thought it was funny, I'll change it! context https://github.com/celestiaorg/celestia-core/pull/1279#discussion_r1548794230

Reply:
renamed as the inside joke got boring